daemon/daemon_windows.go

```diff
@@ -649,3 +654,42 @@ func setupResolvConf(config *config.Config) {
 func (daemon *Daemon) RawSysInfo() *sysinfo.SysInfo {
 	return sysinfo.New()
 }
+
+func (daemon *Daemon) initLibcontainerd(ctx context.Context) error {
+	var err error
+
+	rt := daemon.configStore.GetDefaultRuntimeName()
+	if rt == "" {
+		if daemon.configStore.ContainerdAddr == "" {
```

Comment:
No cases where we use a default value for the containerd address or anything like that?  ISTR some talk about trying to default to V2 on 2022+? (I guess that ship sailed?)

Reply:
So my plan is to follow-up with a PR to embed containerd and use that if no address is specified on both linux and windows.

Reply:
That said, even on Linux we don't default a socket here.
On Windows we just don't start a containerd process like we do on Linux.

Reply:
Ahhh, got it; thanks for the added detail.